tests/App/config.yml

```diff
@@ -77,3 +77,10 @@ security:
             provider: chain_provider
     access_control:
         - { path: ^/, roles: IS_AUTHENTICATED_ANONYMOUSLY }
+
+services:
+    _defaults:
+        autowire: true
+        autoconfigure: true
+    Liip\FunctionalTestBundle\Tests\App\DataFixtures\ORM\:
```

Comment:
I think tests are failing because the service on this resource import should be tagged as fixture. 

```
Liip\FunctionalTestBundle\Tests\App\DataFixtures\ORM\:
    resource: 'DataFixtures/ORM/*'
    tags: ['doctrine.fixture.orm']
```

see http://symfony.com/doc/current/bundles/DoctrineFixturesBundle/index.html#loading-fixtures

Reply:
created https://github.com/magnetik/LiipFunctionalTestBundle/pull/1 for you

Reply:
humm, I was thinking that the "autoconfigure" should have taken care of this automagically (as test are implementing the right interface)

Reply:
I am not sure if DoctrineFixturesBundle did add `__instanceof`, but you might be right. Checking now if they did..

https://symfony.com/blog/new-in-symfony-3-3-service-autoconfiguration

Reply:
You are right, implementing `ORMFixtureInterface` should be enough to autoconfigure the `doctrine.fixture.orm` tag, whereafter `FixturesCompilerPass` will add the fixture to the definition of `doctrine.fixtures.loader`.

* https://github.com/doctrine/DoctrineFixturesBundle/blob/master/DependencyInjection/DoctrineFixturesExtension.php
* https://github.com/doctrine/DoctrineFixturesBundle/blob/master/DependencyInjection/CompilerPass/FixturesCompilerPass.php

Reply:
I've added the tag definition in the `config.yml` and the test are passing. Is that something that can be related to order in which the services are loaded ?

Reply:
Then I guess the `DoctrineFixturesBundle` is not loaded / included with the `test` environment's `AppKernel`? 

That would also explain why it's working in your dev environment, as you probably did include the bundle in your kernel over there.